Packs/CarbonBlackEnterpriseEDR/Integrations/CarbonBlackEnterpriseEDR/CarbonBlackEnterpriseEDR.py

```diff
@@ -0,0 +1,1321 @@
+
+        elif demisto.command() == 'cb-eedr-watchlist-alerts-enable':
+            return_results(enable_watchlist_alert_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-watchlist-alerts-disable':
+            return_results(disable_watchlist_alert_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-watchlist-create':
+            return_results(create_watchlist_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-watchlist-delete':
+            return_results(delete_watchlist_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-watchlist-update':
+            return_results(update_watchlist_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-report-get':
+            return_results(get_report_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-ioc-ignore-status':
+            return_results(get_ignore_ioc_status_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-ioc-ignore':
+            return_results(ignore_ioc_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-ioc-reactivate':
+            return_results(reactivate_ioc_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-report-create':
+            return_results(create_report_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-report-ignore':
+            return_results(ignore_report_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-report-reactivate':
+            return_results(reactivate_report_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-report-ignore-status':
+            return_results(get_report_ignore_status_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-report-remove':
+            return_results(remove_report_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-report-update':
+            return_results(update_report_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-file-device-summary':
+            return_results(get_file_device_summary(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-get-file-metadata':
+            return_results(get_file_metadata_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-files-download-link-get':
+            return_results(get_file_command(client, demisto.args()))
+
+        elif demisto.command() == 'cb-eedr-file-paths':
+            return_results(get_file_path_command(client, demisto.args()))
+
+    # Log exceptions
+    except Exception as e:
+        err_msg = str(e)
+        if 'MALFORMED_JSON' in err_msg:
+            message = err_msg.split('\n')
+            bad_field = json.loads(message[1]).get('field')
```

Comment:
this could raise exception for a few reasons
lets wrap the condition content in try-except

Reply:
Done.
And for the second error message https://github.com/demisto/content/pull/6961#issuecomment-634575983
I made it mandatory in the yml.